hash/megadriv.xml

```diff
@@ -159,6 +167,7 @@ Info on Sega chip labels (from Sunbeam / Digital Corruption)
 		<year>1995</year>
 		<publisher>Viacom New Media</publisher>
 		<info name="serial" value="T-139066-50"/>
+		<sharedfeat name="compatibility" value="PAL"/>  <!-- Region Incompatible -->
```

Comment:
`region locked` rom displays a useful message indicating the game has intentionally been prevented from working on this region (or regions) of console.

`region incompatible` there is no message, game hangs either before or after Sega message, no idea if it is intentional or not.

The third `region issues` for when the game runs, but with serious issues making it unplayable.

It would be a shame to lose this extra information I feel?

Reply:
If the SW explicitly checks region lock and punts with a "for use in x region" message then compatibility should be set to the necessary flag, it's otherwise not a good idea to lock it out, given that MD is very lenient that regard, cfr. [Sega-16 thread](https://www.sega-16.com/forum/showthread.php?10948-GUIDE-Import-gaming-on-your-Mega-Drive-Genesis). I also don't excessively trust our existing codebase so if there's a bug this will eventually backfire.

If you feel that "region incompatible" / "region issues" are worth to be added then I suggest you to use the `notes` field in XML instead, where you describe symptom for each game tested, for example:

```xml
		<notes><![CDATA[
region incompatible: black screen after Sega logo
]]></notes>
```

Reply:
Removed comments as suggested

Reply:
compatibility is only set on the 345 (of the 2508) Roms that won't run at all on those particular incorrect mismatched console regions